packages/file_selector/file_selector/pubspec.yaml

```diff
@@ -12,7 +12,7 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
-  file_selector_platform_interface: ^2.0.0
+  file_selector_platform_interface: ^2.0.2
```

Comment:
This should be resolved by Pub. People who want this feature just need to `flutter clean` and then `flutter build` again, don't they?

Reply:
You're right. Marking this as closed. Thanks @ditman